app/assets/javascripts/profile.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
we do not need any more view or js files in backend. so completely remove all profile coffee, scss ... files